FirebasePhoneAuthUI/CountryCode/FUIPhoneNumber.m

```diff
@@ -0,0 +1,119 @@
+//
+//  Copyright (c) 2016 Google Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+#import "FUIPhoneNumber.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+NSString * const FUIPhoneNumberValidationErrorDomain = @"FUIPhoneNumberValidationErrorDomain";
+
+@implementation FUIPhoneNumber
+
+- (nullable instancetype)initWithNormalizedPhoneNumber:(NSString *)normalizedPhoneNumber {
```

Comment:
This function takes a nonnull `normalizedPhoneNumber`, so maybe instead of returning `nil` we can assert and return a nonnull type as well.